internal/backend/db/dbgorm/connections.go

```diff
@@ -15,10 +15,10 @@ func (db *gormdb) createConnection(ctx context.Context, conn *scheme.Connection)
 
 func (db *gormdb) CreateConnection(ctx context.Context, conn sdktypes.Connection) error {
 	c := scheme.Connection{
-		ConnectionID:     conn.ID().String(),
-		IntegrationID:    scheme.PtrOrNil(conn.IntegrationID().String()), // TODO(ENG-158): need to verify integration id
+		ConnectionID:     *conn.ID().UUIDValue(),
+		IntegrationID:    conn.IntegrationID().UUIDValue(),
```

Comment:
you removed TODO comment which is still relevant

Reply:
correct, restored